docs/configuration.md

```diff
@@ -0,0 +1,158 @@
+
+The schema follows the rules defined in http://json-schema.org/.
+The Dapplet settings UI is generated by [react-jsonschema-form](https://react-jsonschema-form.readthedocs.io/en/latest/usage/single/).
+
+In the production environment, we set up the **dapplet's config**, and specify the parameters that will be available in the browser through the extension.
+
+:::caution
+
+It is possible to edit and create a **dapplet's config** only from the working environment of the module in the extension, it is possible to use the **dapplet's config**, depending on how the developer configured them.
+
+:::
+
+
+### Configuration
+
+The **dapplet's config** is located in the */config* folder.
+
+In the `schema.json` file, specify the dapplet settings that will be available to the user.
+
+The scheme consists of parameters:
+
+  ```js
+ {
+// schema type: always object
+  "type": "object",
+// the fields you want to specify in your schema
+  "required": [
+   ...
+  ],
+// field properties
+  "properties": {
+   ...
+  }
+}
+  ```
+
+![img](/img/con_01.png)
+
+In the *"properties"* we specify the parameters for each field, also in the form of objects.
+
+  ```js
+ "properties": {
+    // field's name
+    "network": {
+        // field's type: string or number
+      "type": ...,
+        // the title of the field that the user sees
+      "title": ...,
+        // optional field visibility parameter. Boolean's value
+      "hidden": ...,
+        // optional field in select format  
+      "enum": [
+        ...
+      ]
+    },
+    }
+  ```
+
+![img](/img/con_02.png)
+
+  Fields in the format for entering numeric values with a maximum and minimum value limit and a step.
+
+ ```js
+{
+  "type": "object",
+  "required": [
+    ...
+  ],
+  "properties": {
+    "network": {
+      "type": "string",
+      "title": ...,
+      "enum": [
+       ...
+      ]
+    },
+    "step": {
+      "type": "number",
+      "title": ...,
+    // maximum value
+      "maximum": ...,
+    // minimum value
+      "minimum": ...,
+    // The value by which the data changes when using the widget (in this example, clicking on the arrow)
+      "multipleOf": ...,
+    // Step by which the value changes
+      "step": ...
+    },
+  }
+}
+  ```
+
+![img](/img/con_04.png)
+![img](/img/con_03.png)
+
+The default setting values are defined in the `default.json`.
+
+There are three environments:
+
+- `dev` - used when a module is loaded from the development server;
+- `test` - used when a module is loaded from the Test Dapplet Registry;
+- `prod` or `main` - used when a module is loaded from the Production Dapplet Registry.
+
+Describing fields as an object:
+
+  ```js
+{
+    // designate the development environment
+  "main": {
+    //  the name of the field that is specified in the **required** `schema.json`
+    "network": ..., // field's value
+    "step": ...,
+    "delay": ...
+  },
+  "test": {
+  ...
+  },
+  "dev": {
+  ...
+  }
+}
+  ```
+
+![img](/img/con_05.png)
```

Comment:
Also don't use the picture. Show this values in the code block above. And the network specify with the testnet for the dev mode.